datadriven.go

```diff
@@ -36,8 +36,18 @@ var (
 			"run. Used to update tests when a change affects many cases; please verify the testfile "+
 			"diffs carefully!",
 	)
+
+	traceLog = flag.Bool(
+		"trace", false,
+		"echo the directives and responses from test files.",
+	)
 )
 
+// Verbose returns true iff -trace was passed.
+func Verbose() bool {
```

Comment:
Do we need this?

Reply:
Yes - I am using it in another PR which I will push your way in a few minutes